### PR TITLE
Leaderboard

### PR DESCRIPTION
**Pull request by kevashans**
- Adjusted Leaderboard design to match Figma design
- Created base navigation structure
- Created base User class design

### CODE REVIEW

**Review by gursimar03 — APPROVED** [review]
Good work.

File: Mobile App/app/src/main/java/com/example/scar/MainScreen.kt
Comment (almezj) [inline Mobile App/app/src/main/java/com/example/scar/MainScreen.kt:1]: Needs import statement formatting

File: Mobile App/app/src/main/java/com/example/scar/ui/theme/RegionalLeaderboard.kt
Comment (almezj) [inline Mobile App/app/src/main/java/com/example/scar/ui/theme/RegionalLeaderboard.kt:1]: Needs import statement formatting

File: Mobile App/app/src/main/java/com/example/scar/MainScreen.kt
Comment (almezj) [inline Mobile App/app/src/main/java/com/example/scar/MainScreen.kt:1]: Needs import statement formatting

File: .idea/modules.xml
Comment (almezj) [inline .idea/modules.xml:1]: Gitignore .idea folder

File: .idea/vcs.xml
Comment (almezj) [inline .idea/vcs.xml:1]: Gitignore .idea folder

File: .idea/workspace.xml
Comment (almezj) [inline .idea/workspace.xml:1]: Gitignore .idea folder

**Review by almezj — CHANGES_REQUESTED** [review]

**Review by almezj — APPROVED** [review]

**Review by v-philip — APPROVED** [review]
Good work!

**Pull request merged**